src/wasm/body.rs

```diff
@@ -44,6 +44,13 @@ impl Body {
             inner: Inner::Multipart(f),
         }
     }
+
+    pub(crate) fn is_empty(&self) -> bool {
+        match &self.inner {
+            Inner::Bytes(bytes) => bytes.is_empty(),
+            Inner::Multipart(form) => form.is_empty(),
```

Comment:
I'm unsure whether this is actually correct, and don't know whether it's useful to return `true` for multipart forms with 0 fields here, but it made sense to me 🤷